src/plugins/unified_search/public/search_bar/create_search_bar.tsx

```diff
@@ -193,6 +194,9 @@ export function createSearchBar({
         true
       );
     }, [query, timeRange, useDefaultBehaviors]);
+    const showSaveQuery =
+      !isOfAggregateQueryType(query) &&
+      (Boolean(core.application.capabilities.savedQueryManagement?.edit) || props.showSaveQuery);
```

Comment:
@davismcphee that's how it works to apply the change on unified search level. 
With this the change in Discover of this PR is redundant.

Reply:
Make sense to me! This way is better than the old way anyway since it bakes the ES|QL check directly into Unified Search without needing consumers to know or care 👍